[Bug] Upstream Sync workflow fails for all forks due to actions-cool/issues-helper being disabled by GitHub

### 📱 Client Type

Other

### 💻 Operating System

Other

### 📦 Deployment Platform

Other

### 📌 Version

v2.2.1-canary.15

### 🌐 Browser

Other

### 🐛 What happened?

The scheduled "Upstream Sync" workflow at `.github/workflows/sync.yml` has been failing for ALL fork repositories since May 19, 2026.

**Error**: `Repository access blocked` at the "Getting action download info" phase.

**Root Cause**: The workflow references `actions-cool/issues-helper@e361abf` (used for opening/closing sync failure issues). On May 19, 2026, this action was compromised in a supply chain attack — attackers moved all tags to point to malicious commits that exfiltrate CI/CD credentials. GitHub has since permanently disabled the repository:
https://github.com/actions-cool/issues-helper

Because the repo is disabled, GitHub Actions cannot download the action, causing ALL fork sync workflows to fail immediately.

### 📷 How to reproduce it?

1. Fork the `lobehub/lobehub` repository (or any existing fork)
2. Wait for the scheduled "Upstream Sync" workflow to trigger (every 2 hours)
3. Or manually trigger via `workflow_dispatch` on the Actions tab
4. Observe: workflow fails at initialization with `##[error]Repository access blocked`

### 🚦 What it should be?

Replace `actions-cool/issues-helper` with GitHub CLI commands or a maintained alternative.

**Suggested fix** — replace the issue-helper steps with `gh` CLI:

```yaml
- name: Clean issue notice
  env:
    GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
  run: |
    gh issue list --label "🚨 Sync Fail" --state open --json number -q '.[].number' |
    while read num; do gh issue close "$num"; done

- name: Sync check
  if: failure()
  env:
    GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
  run: |
    gh issue create \
      --title "🚨 同步失败 | Sync Fail" \
      --label "🚨 Sync Fail" \
      --body "..."
```

**References**:
- Attack report: https://thehackernews.com/2026/05/github-actions-supply-chain-attack.html
- StepSecurity analysis: https://www.stepsecurity.io/blog/actions-cool-issues-helper-github-action-compromised

### 📝 Additional Information

**Important note**: Because the workflow uses a SHA-pinned reference (`e361abf...`), no malicious code would have executed. However, since GitHub entirely disabled the repo, the action cannot be downloaded at all — hence the "Repository access blocked" error.

The `actions-cool/maintain-one-comment` action was also compromised in the same attack. Please check if it's used elsewhere in this repo.

This issue affects EVERY fork of lobehub/lobehub. The upstream sync workflow is the primary way forks stay updated, so this is a high-priority fix.

### 🛠️ Willing to Submit a PR?

None

### ✅ Validations

- [x] Read the [docs](https://lobehub.com/zh/docs).
- [x] Check that there isn't [already an issue](https://github.com/lobehub/lobe-chat/issues) that reports the same bug to avoid creating a duplicate.
- [x] Make sure this is a LobeChat issue and not a third-party library or provider issue.
- [x] Check that this is a concrete bug. For Q&A, please use [GitHub Discussions](https://github.com/lobehub/lobe-chat/discussions) or join our [Discord Server](https://discord.gg/rGHwKq4R).

--- a/.github/workflows/sync.yml
+++ b/.github/workflows/sync.yml
@@ -19,12 +19,6 @@ jobs:
     steps:
       - uses: actions/checkout@v6
 
-      - name: Clean issue notice
-        uses: actions-cool/issues-helper@e361abf610221f09495ad510cb1e69328d839e1c # v3.7.6
-        with:
-          actions: 'close-issues'
-          labels: '🚨 Sync Fail'
-
       - name: Sync upstream changes
         id: sync
         uses: aormsby/Fork-Sync-With-Upstream-action@v3.4
@@ -34,21 +28,3 @@ jobs:
           target_sync_branch: main
           target_repo_token: ${{ secrets.GITHUB_TOKEN }} # automatically generated, no need to set
           test_mode: false
-
-      - name: Sync check
-        if: failure()
-        uses: actions-cool/issues-helper@e361abf610221f09495ad510cb1e69328d839e1c # v3.7.6
-        with:
-          actions: 'create-issue'
-          title: '🚨 同步失败 | Sync Fail'
-          labels: '🚨 Sync Fail'
-          body: |
-            Due to a change in the workflow file of the [LobeChat][lobechat] upstream repository, GitHub has automatically suspended the scheduled automatic update. You need to manually sync your fork. Please refer to the detailed [Tutorial][tutorial-en-US] for instructions.
-
-            由于 [LobeChat][lobechat] 上游仓库的 workflow 文件变更，导致 GitHub 自动暂停了本次自动更新，你需要手动 Sync Fork 一次，请查看 [详细教程][tutorial-zh-CN]
-
-            ![](https://github-production-user-asset-6210df.s3.amazonaws.com/17870709/273954625-df80c890-0822-4ac2-95e6-c990785cbed5.png)
-
-            [lobechat]: https://github.com/lobehub/lobe-chat
-            [tutorial-zh-CN]: https://lobehub.com/zh/docs/self-hosting/advanced/upstream-sync
-            [tutorial-en-US]: https://lobehub.com/docs/self-hosting/advanced/upstream-sync

